plugins/generator-1.17.1/forge-1.17.1/procedures/entity_lookpos_x.java.ftl

```diff
@@ -0,0 +1,6 @@
+(new Object() {
+    public int getLookX(Entity _ent, double maxDist) {
+        return _ent.level.clip(new ClipContext(_ent.getEyePosition(1f), _ent.getEyePosition(1f).add(_ent.getViewVector(1f).scale(maxDist)),

```

Comment:
Since this is a single statement, I don't think we need an inline Object class in these cases as this Object thingy is an ugly hack we should avoid at all costs where possible